ictt-erc20/contract/Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: help install build test clean deploy-fuji initialize-fuji deploy-dispatch initialize-dispatch register-remote mint send balance deploy-and-send
+.PHONY: help install build test clean deploy-home deploy-remote initialize-home initialize-remote register-remote mint send balance-home balance-remote deploy-and-send fmt
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)